_posts/2021-04-06-WMD-2021-results.md

```diff
@@ -0,0 +1,304 @@
+   </td>
+   <td>pull requests opened
+   </td>
+   <td>issues closed
+   </td>
+   <td>issues opened
+   </td>
+  </tr>
+  <tr>
+   <td><a href="http://moveit.ros.org">moveit.ros.org</a>
+   </td>
+   <td><p style="text-align: right">
+18</p>
+
+   </td>
+   <td><p style="text-align: right">
+20</p>
+
+   </td>
+   <td><p style="text-align: right">
+0</p>
+
+   </td>
+   <td><p style="text-align: right">
+2</p>
+
+   </td>
+  </tr>
+  <tr>
+   <td>moveit_tutorials
+   </td>
+   <td><p style="text-align: right">
+20</p>
+
+   </td>
+   <td><p style="text-align: right">
+22</p>
+
+   </td>
+   <td><p style="text-align: right">
+3</p>
+
+   </td>
+   <td><p style="text-align: right">
+6</p>
+
+   </td>
+  </tr>
+  <tr>
+   <td>moveit2_tutorials
+   </td>
+   <td><p style="text-align: right">
+11</p>
+
+   </td>
+   <td><p style="text-align: right">
+17</p>
+
+   </td>
+   <td><p style="text-align: right">
+2</p>
+
+   </td>
+   <td><p style="text-align: right">
+2</p>
+
+   </td>
+  </tr>
+  <tr>
+   <td>moveit
+   </td>
+   <td><p style="text-align: right">
+14</p>
+
+   </td>
+   <td><p style="text-align: right">
+20</p>
+
+   </td>
+   <td><p style="text-align: right">
+12</p>
+
+   </td>
+   <td><p style="text-align: right">
+21</p>
+
+   </td>
+  </tr>
+  <tr>
+   <td>moveit2
+   </td>
+   <td><p style="text-align: right">
+24</p>
+
+   </td>
+   <td><p style="text-align: right">
+22</p>
+
+   </td>
+   <td><p style="text-align: right">
+10</p>
+
+   </td>
+   <td><p style="text-align: right">
+11</p>
+
+   </td>
+  </tr>
+  <tr>
+   <td>moveit_msgs
+   </td>
+   <td><p style="text-align: right">
+7</p>
+
+   </td>
+   <td><p style="text-align: right">
+6</p>
+
+   </td>
+   <td><p style="text-align: right">
+0</p>
+
+   </td>
+   <td><p style="text-align: right">
+0</p>
+
+   </td>
+  </tr>
+  <tr>
+   <td>moveit_task_constructor
+   </td>
+   <td><p style="text-align: right">
+5</p>
+
+   </td>
+   <td><p style="text-align: right">
+6</p>
+
+   </td>
+   <td><p style="text-align: right">
+15</p>
+
+   </td>
+   <td><p style="text-align: right">
+7</p>
+
+   </td>
+  </tr>
+  <tr>
+   <td>moveit_grasps
+   </td>
+   <td><p style="text-align: right">
+0</p>
+
+   </td>
+   <td><p style="text-align: right">
+0</p>
+
+   </td>
+   <td><p style="text-align: right">
+0</p>
+
+   </td>
+   <td><p style="text-align: right">
+0</p>
+
+   </td>
+  </tr>
+  <tr>
+   <td>moveit_visual_tools
+   </td>
+   <td><p style="text-align: right">
+0</p>
+
+   </td>
+   <td><p style="text-align: right">
+0</p>
+
+   </td>
+   <td><p style="text-align: right">
+0</p>
+
+   </td>
+   <td><p style="text-align: right">
+1</p>
+
+   </td>
+  </tr>
+  <tr>
+   <td><strong>Total</strong>
+   </td>
+   <td><p style="text-align: right">
+99</p>
+
+   </td>
+   <td><p style="text-align: right">
+113</p>
+
+   </td>
+   <td><p style="text-align: right">
+42</p>
+
+   </td>
+   <td><p style="text-align: right">
+50</p>
+
+   </td>
+  </tr>
+</table>
```

Comment:
I strongly prefer [tables in markdown format](https://github.github.com/gfm/#tables-extension-) over raw HTML.